app/controllers/application_controller.rb

```diff
@@ -43,9 +39,7 @@ def create_guest
   end
 
   def save_guest
-    guest_user_name="guest"+(0...rand(25)).map { (65 + rand(26)).chr }.join
-    guest_email="guest"+(0...rand(25)).map { (65 + rand(26)).chr }.join + "@unknown.com"
-    user= User.create(user_name:guest_user_name,email:guest_email)
+    user= User.new(user_name:"guest#{(0...rand(25)).map { (65 + rand(26)).chr }.join}",email:"guest#{(0...rand(25)).map { (65 + rand(26)).chr }.join}@unknown.com")
     user.save(validate: false)
```

Comment:
Make a method in User model and use that here.

Reply:
Done